src/emailpro/account/update/emailpro-account-update.controller.js

```diff
@@ -1,242 +1,295 @@
-angular.module('Module.emailpro.controllers').controller('EmailProUpdateAccountCtrl', ($q, $scope, $stateParams, $translate, EmailPro, EmailProPassword) => {
-  const originalValues = angular.copy($scope.currentActionData);
-
-  const accountIsValid = function accountIsValid() {
-    const account = $scope.selectedAccount;
-    if ($scope.simplePasswordFlag
-      || $scope.differentPasswordFlag
-      || $scope.containsNameFlag) {
-      return false;
-    } if (account && /\s/.test(account.password)) {
-      return false;
-    } if (!account.canBeConfigured && !account.password) {
-      return false;
-    } if (!account.domain || !account.login) {
-      return false;
+angular.module('Module.emailpro.controllers').controller(
+  'EmailProUpdateAccountCtrl',
+  class EmailProUpdateAccountCtrl {
+    constructor(
```

Comment:
Could be a nice enhancement;)
```suggestion
    /* @ngInject */
    constructor(
```